plugins/native/configs/config.d/10_native.json

```diff
@@ -62,6 +62,26 @@
       "translucency-support": false
     }
   },
+  {
+    "match": ".*(spz)",
+    "options": {
+      "point-sprites": true,
+      "point-sprites-size": 1,
+      "point-sprites-type": "gaussian",
+      "scalar-coloring": true,
+      "coloring-array": "color",
+      "coloring-component": "-2",
+      "background-color": "0, 0, 0",
+      "up": "Y",
+      "interaction-trackball": true,
+      "camera-position": "0,1,4",
```

Comment:
any specific reason for that ?

Reply:
works well with samples provided by Niantic